package.json

```diff
@@ -34,7 +34,14 @@
   },
   "resolutions": {
     "@types/react": "^16.14.25",
-    "@types/react-dom": "^16.9.14"
+    "@types/react-dom": "^16.9.14",
+    "@storybook/addons": "^6.5.0-beta.8",
+    "webpack": "^5.72.0",
+    "@storybook/react/webpack": "^5.72.0",
+    "@storybook/core-common/webpack": "^5.72.0",
+    "@storybook/core-server/webpack": "^5.72.0",
+    "@storybook/builder-webpack4/webpack": "^5.72.0",
```

Comment:
is it for webpack5 compatibility?

Reply:
is it still needed if we set it to webpack 5 in DS?
https://github.com/Talend/ui/blob/732924abac8e5ea66ed025d085d6f8b6c9956153/packages/design-tokens/.storybook/main.js#L8

Reply:
yes indeed. storybook do not want to create a 7.0 version so they let webpack 4 as a dependency ...
We do not have other choice than set the builder and force resolution. Which means all project will have to do so.
Quite disappointed also by their choice.